app/models/mixins/miq_web_server_worker_mixin.rb

```diff
@@ -105,12 +105,12 @@ def self.sync_workers
       result
     end
 
-    def self.thin_pid_file(port)
-      Rails.root.join("tmp/pids/thin.#{port}.pid")
+    def self.pid_file(port)
+      Rails.root.join("tmp/pids/puma.#{port}.pid")
```

Comment:
@jrafanie @gtanzillo done! i like that better too. creating an accessor didn't seem to be necessary in this case. here's what i got after spinning up:

```
tim@tim-thinkpad:~/src/manageiq$ ps aux | grep ruby | grep puma
tim       7804 98.0  0.3 151284 59428 pts/0    RNl  13:57   0:01 ruby /home/tim/src/manageiq/bin/rails server puma --port 3000 --binding 0.0.0.0 --environment development --config /home/tim/src/manageiq/config.ru --pid /home/tim/src/manageiq/tmp/pids/rails_server.3000.pid
tim       7808 78.5  0.3 139876 50040 pts/0    RNl  13:57   0:01 ruby /home/tim/src/manageiq/bin/rails server puma --port 4000 --binding 0.0.0.0 --environment development --config /home/tim/src/manageiq/config.ru --pid /home/tim/src/manageiq/tmp/pids/rails_server.4000.pid
```